snorkeling_helper/label_functions/knowledge_bases/hetionet_knowledge_base_generator.py

```diff
@@ -0,0 +1,380 @@
+# ---
+# jupyter:
+#   jupytext:
+#     formats: ipynb,py
+#     text_representation:
+#       extension: .py
+#       format_name: light
+#       format_version: '1.5'
+#       jupytext_version: 1.9.1+dev
+#   kernelspec:
+#     display_name: Python [conda env:snorkeling_full_text]
+#     language: python
+#     name: conda-env-snorkeling_full_text-py
+# ---
+
+# # Disease Associates Genes Edge Prediction
+
+# This notebook is designed to take the next step moving from predicted sentences to edge predictions. After training the discriminator model, each sentences contains a confidence score for the likelihood of mentioning a relationship. Multiple relationships contain multiple sentences, which makes establishing an edge unintuitive. Is taking the max score appropriate for determining existence of an edge? Does taking the mean of each relationship make more sense? The answer towards these questions are shown below.
+
+# +
+# %load_ext autoreload
+# %autoreload 2
+# %matplotlib inline
+
+from pathlib import Path
+
+import pandas as pd
+import plydata as ply
+import plydata.tidy as ply_tdy
+from sqlalchemy import create_engine
+
+# +
+# Set up the environment
+username = "danich1"
+password = "snorkel"
+dbname = "pubmed_central_db"
```

Comment:
Looks like just test credentials and might be safe to make them publicly, but another option could be using environmental variables to avoid leaving this here.

Reply:
>Looks like just test credentials and might be safe to make them publicly, but another option could be using environmental variables to avoid leaving this here.

These are the credentials to log onto the local database I have. Normally, I'd agree about hiding sensitive information; however, everything in this database is publicly available. So if someone were to "hack" into this database they would only get information that everyone could see anyways. Will keep this note in mind moving forward on projects with more sensitive data.